components/rule-mgt/org.wso2.carbon.identity.rule.metadata/src/test/java/org/wso2/carbon/identity/rule/metadata/config/FlowConfigTest.java

```diff
@@ -133,6 +133,18 @@ private Map<String, FieldDefinition> getMockedFieldDefinitions() {
         Value grantTypeValue = new OptionsInputValue(Value.ValueType.STRING, optionsValues);
         fieldDefinitionMap.put("grantType", new FieldDefinition(grantTypeField, operators, grantTypeValue));
 
+        Field flowField = new Field("flow", "flow");
+        List<OptionsValue> flowOptionsValues = Arrays.asList(
+                new OptionsValue("admin_initiated_password_reset", "Admin initiated password reset"),
```

Comment:
So far I think we have used camel case. Can't we stick with that here. So it's consistent

Reply:
Fixed